src/main/java/frc/robot/subsystems/swerve/modules/drive/talonfx/TalonFXDriveConfigObject.java

```diff
@@ -2,17 +2,22 @@
 
 import com.ctre.phoenix6.BaseStatusSignal;
 import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import edu.wpi.first.wpilibj.Timer;
 import frc.robot.constants.GlobalConstants;
 import frc.robot.poseestimation.PoseEstimatorConstants;
 import frc.utils.ctre.CTREDeviceID;
+import frc.utils.ctre.PhoenixProUtils;
 import frc.utils.devicewrappers.TalonFXWrapper;
+import org.littletonrobotics.junction.Logger;
 
 class TalonFXDriveConfigObject {
 
 	private final TalonFXWrapper motor;
 	private final TalonFXDriveSignals signals;
+	private final String logPath;
 
-	protected TalonFXDriveConfigObject(CTREDeviceID motorID, boolean inverted, TalonFXConfiguration configuration) {
+	protected TalonFXDriveConfigObject(String logPathPrefix, CTREDeviceID motorID, boolean inverted, TalonFXConfiguration configuration) {
+		this.logPath = logPathPrefix + "ConfigObject/";
```

Comment:
no need  "ConfigObject/" because i want to know which module and what fail and it doesnt give more information